src/lib.rs

```diff
@@ -0,0 +1,16 @@
+//! Approximate Lower Bound Arguments (ALBA, <https://eprint.iacr.org/2023/1655.pdf>)
+//!
+//! Alba is a generic protocol to prove succinctly a lower bound of the size of
+//! a, potentially weighted, set. Say we have a set Sp of size |Sp| >= $n_p$,
+//! and a lower bound $n_f$ < $n_p$ of it we want to prove. Alba gives us a
+//! method to generate a proof of knowledge of this bound by finding the
```

Comment:
`the smallest` - is it unique? Is it really smaller in the sense that it cannot contain a smaller set that also convinces a verifier ?